### DISCUSSION TITLE
Is the Docker image build pipeline publicly available?

### DISCUSSION THREAD

**Discussion by MrShiister** [Keyfactor/ejbca-ce#328] [Q&A]
I am forking the EJBCA repository to make my modifications, with intention to then build my own EJBCA Docker image.

Is the build pipeline available to the public? If not, can I at least know the build process? Knowing the pipeline process, or at least how it works, would be greatly helpful.

Thank you so much!

**Comment by primetomas** [comment]
The pipe-line is not available unfortunately. The software components are pretty basic, it's the startup scripts that makes the experience smooth.
